src/Support/helpers.php

```diff
@@ -24,27 +24,27 @@ function auth()
     }
 }
 
-if (! function_exists('schema')) {
+if (! function_exists('types')) {
     /**
      * Get instance of schema container.
      *
-     * @return \Nuwave\Lighthouse\Schema\SchemaBuilder
+     * @return \Nuwave\Lighthouse\Schema\TypeRegistry
      */
-    function schema()
+    function types()
```

Comment:
I think `schema` may be a safer global function name (but that's just a personal preference)... let's add the `schema` function back and have it just call the `types` function to minimize breaking changes.

Reply:
I can rename it to schema, no problem. The global functions do seem a bit dangerous though. Maybe, in the next major release, we can namespace them under graphql() ?

Reply:
@spawnia totally agree w/ that, much safer